Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -160,7 +160,11 @@ void DMainWindow::DoStartPause()
 
 void DMainWindow::OnOpen()
 {
-	StartGame(ShowFileDialog());
+	QString filename = ShowFileDialog();
+	if (filename.toStdString() != "")
```

Comment:
Well, I'm not quite sure. Aren't there some characters `std::string` can't hold?

Of course, in such a case we should warn the user...